kivymd/uix/behaviors/elevation.py

```diff
@@ -183,37 +171,98 @@ class CommonElevationBehavior(object):
     """
 
     _elevation = NumericProperty(0)
+
+    # soft shadow
     _soft_shadow_texture = ObjectProperty()
     _soft_shadow_size = ListProperty((0, 0))
     _soft_shadow_pos = ListProperty((0, 0))
+    _soft_shadow_cl = ListProperty([1, 1, 1, 0])
     _soft_shadow_a = NumericProperty(0)
+
+    # hard shadow
     _hard_shadow_texture = ObjectProperty()
     _hard_shadow_size = ListProperty((0, 0))
     _hard_shadow_pos = ListProperty((0, 0))
+    _hard_shadow_cl = ListProperty([1, 1, 1, 0])
     _hard_shadow_a = NumericProperty(0)
 
     def __init__(self, **kwargs):
+        #
+        im = BytesIO()
+        Image.new("RGBA", (4, 4), color=(0, 0, 0, 0)).save(im, format="png")
+        im.seek(0)
+        #
+        self._soft_shadow_texture = self._hard_shadow_texture = CoreImage(
+            im, ext="png"
+        ).texture
+        super().__init__(**kwargs)
+        Clock.schedule_once(self.shadow_preset, -1)
```

Comment:
This section now uses a pre generated image of 4x4 alpha pixels that will be contained in memory until the real texture is load in memory.
this is  the base to generate better shadows for elevated widgets.